Sources/KituraContracts/CodableQuery/QueryEncoder.swift

```diff
@@ -46,6 +46,18 @@ public class QueryEncoder: Coder, Encoder {
         return "?" + String(desc.dropFirst())
     }
 
+    /// Encodes an Encodable object to a URLQueryItem array
```

Comment:
What's this code?

Reply:
That's not something I'd added, it's already in master. Not sure why that got shown as changed.